workshops/3d_models_with_zdog/README.md

```diff
@@ -0,0 +1,250 @@
+
+```javascript
+    const cube = new Zdog.Box({
+      addTo: ws,
+      width: 100,
+      height: 100,
+      depth: 100,
+      stroke: false,
+      leftFace: "#da0",
+      rightFace: "#e62",
+      topFace: '#ed0',
+      bottomFace: '#636'
+    })
+```
+Explanation:
+1. Box is a shape class, you can replace this for the shape you want to use…
+2. We made the 3d model (cube) a child to the main Zdog Illustration (ws) using the addTo element. This element must be there, otherwise the 3d model will not render.
+3. The width, height and depth elements can either stretch or shrink the shape of your box:
+- Width: Sets the width of the cube
+- Height: Sets the height of the cube
+- Depth: Sets the depth of the cube. If the value is 0, the cube will render as a 2d square; so make sure to give it a value.
+4. The Stroke element gives the 3d model a stroke, it works as an external layer that you can use to give your 3d model a rounded look. Play around with it!
+5. leftFace, rightFace, topFace & bottomFace elements give a color to each face of the 3d model, try using different colors for each face, so that you can appreciate way better the animations you make.
+
+### Rendering
+
+Now that we have created our 3d model, let’s render it. Use the following line of code to render the cube we just created. 
+
+```javascript
+ws.updateRenderGraph()
+```
+This code updates and render your Zdog illustration that was declared in the first variable, so make sure to write the correct name in.
+
+Now let's click on the **Run** button to see what happens...
+![Render image](https://cloud-k11ck8g2n.vercel.app/0image.png)
+
+Congrats, you just made your first 3d model… Yeah, maybe not what you were expecting. Let’s fix this by animating it.
+
+![woah gif](https://cloud-kr2lyxjbx.vercel.app/0woah.gif)
+
+### Animating it
+Add the following code to our JS file:
+
+```javascript
+    function animatemodel() {
+      ws.rotate.y += 0.01;
+      ws.rotate.x += 0.01;
+      ws.updateRenderGraph()
+      requestAnimationFrame(animatemodel)
+    }
+
+    animatemodel()
+```
+Explanation:
+
+1. We just created a function that will make the 3d model rotate, you can name this function to whatever you want.
+2. ```rotate.x``` and ```rotate.y``` set’s the velocity to the rotation of the model: 
+- The cube will move up and down depending on the given value (- or +, respectively)
+- The cube will move to the right and to the left depeding on the given value (- or +, respectively)
+3. ```ws.updateRenderGraph()``` updates and render your Zdog illustration that was declared in the first variable, make sure to write the correct name in.
+4. ```requestAnimationFrame(animatemodel)``` this is like a loop, basically it makes the model rotates every time by creating frames.
+5. ```animeatemodel()``` calls the function.
+
+Now you can click on run again!
+
+![Animated model](https://cloud-djkuut7y4.vercel.app/0screen_recording_2020-10-03_at_7.31.47_pm.gif)
```

Comment:
:((( this gif doesn't seem to be displaying

Reply:
might just be a thing on my computer
<img width="263" alt="Screenshot 2020-10-14 at 10 31 38 AM" src="https://user-images.githubusercontent.com/39828164/95936665-b4c0e600-0e08-11eb-89d2-bcfdc6782398.png">

Reply:
Should i upload it again to #cdn or nah? @sampoder

Reply:
I guess